packages/generator/src/sdk-metadata/pregenerated-lib.ts

```diff
@@ -7,10 +7,19 @@ import {
   genericDescription,
   s4hanaCloudDescription
 } from '../package-description';
-import { PregeneratedLibrary } from './sdk-metadata-types';
+import type {
+  InstructionWithText,
+  PregeneratedLibrary
+} from './sdk-metadata-types';
 
-export function getInstallationSnippet(service: VdmServiceMetadata): string {
-  return `npm i ${service.npmPackageName}:latest`;
+export function getInstallationSnippet(
+  service: VdmServiceMetadata
+): InstructionWithText {
+  return {
+    instructions: `npm i ${service.npmPackageName}:latest`,
```

Comment:
question: do we only consider npm for the first version?

Reply:
hm you mean yarn? I guess npm is more standard and all yarn users know what to do.

Reply:
From my side only npm is fine. But why do we use `...:latest`?